app/utils/get_filtered_records.py

```diff
@@ -23,10 +25,26 @@ def get_all_records():
                          'isotype_comb', 'academic_primary_estimate', 'dashboard_primary_estimate', 'pop_adj',
                          'test_adj', 'specimen_type', 'test_type', 'population_group', 'url']
 
+        # Add columns from dashboard source to select statement
         fields_list = [DashboardSource.source_id]
         for field_string in field_strings:
             fields_list.append(getattr(DashboardSource, field_string))
 
+        # If research fields is True, add columns from research source to select statement
+        if research_fields:
+            research_source_cols = ['case_population', 'deaths_population', 'age_max', 'age_min', 'age_variation',
```

Comment:
do we want to put this in some config?

Reply:
i didnt put it in a config because the list of fields for dashboard source above was not in a config

Reply:
okay leaving it up to your judgement